firebase-database/src/androidTest/java/com/google/firebase/database/DatabaseRegistrarTest.java

```diff
@@ -0,0 +1,28 @@
+package com.google.firebase.database;
```

Comment:
These should be robolectric tests if possible not to pay the cost of connected tests here.

You may need to add gradle configuration to make it work similar to https://github.com/firebase/firebase-android-sdk/pull/128, specifically around `enableUnitTestBinaryResources` and `includeAndroidResources`

Reply:
Good point! I have pushed up a change that attempts to do this for database. I am seeing that the registrar is not being discovered by firebase app.
I have thrown an exception intentionally inside FirebaseApp. Can I get your help to take a look?